ui/src/pages/HowToPage.js

```diff
@@ -0,0 +1,172 @@
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, {Component} from 'react';
+import {FormattedMessage, defineMessages, injectIntl} from 'react-intl';
+
+import StaticPageWrapper from './../components/StaticPageWrapper.js';
+import Zippy from './../components/Zippy.js';
+
+const MESSAGES = defineMessages({
+  helpDocument: {
+    id: 'HowToPage.helpDocument',
+    defaultMessage: 'Help document',
+    description: 'A header for a page with help documentation.',
+  },
+  intro: {
+    id: 'HowToPage.intro',
+    defaultMessage: ('Person Finder can be used by anyone from a PC or mobile '
```

Comment:
just curious, s/PC/computer?

Reply:
I'll ask about that when we get into this with product folks, but the current language says PC.